GetStream/stream-chat-react-native
# fix(android): various attachment picker issues

## 🎯 Goal

There are many problems with attachment picker on Android that this PR aims to fix:
1. Random popups of the bottom sheet #1283 (need to release this in beta and ask customers to confirm if this is indeed fixed)
2. When gallery permission is denied, the asking for permission error is only shown when the empty bottom sheet is dragged down
3. When gallery permission is denied, "reconnecting..." bar is shown off and removed on and on.. this is because the app goes background to foreground constantly due to requesting of permissions (which shows the android system dialog over the app even transparently)
4. update bottomsheet lib to latest.. as few bug fixes were done compared to the version we currently use.

## 🛠 Implementation details

NA

## 🎨 UI Changes

NA

## 🧪 Testing

- open/close keyboard frequently on android or change android keyboard.. like gif to normal and back
	- bottom sheet may open for a short time but it should stay closed after.. short open is okay due to internal container layout of bottom sheet
- tap message input wait until keyboard opens and then open attachment picker
	- attachment picker should stay open and keyboard should close
- open attachment picker, wait till it opens. Then tap the message input
	- attachment picker should close and the keyboard should open

## ☑️ Checklist
